internal/handlers/organization.go

```diff
@@ -117,6 +116,26 @@ func (api *API) CreateOrganization(c *gin.Context) {
 		}
 		return
 	}
+
+	// Create a default security group for the organization
+	sg, err := api.createDefaultSecurityGroup(ctx, org.ID.String())
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, models.NewApiInternalError(err))
+		return
```

Comment:
https://github.com/nexodus-io/nexodus/issues/1131

This issue also notes that here we end up returning an error from the API, but the org is still created and left in the DB, but in an inconsistent state (without its default security group).